salt/metalk8s/volumes/prepared/files/sparse_volume_cleanup.py

```diff
@@ -0,0 +1,158 @@
+#!/usr/bin/env python2
+"""Script for cleaning up MetalK8s sparse loop volumes.
+
+Handle discovery of the device through a Volume's UUID, and then detach the
+device using the right `ioctl`.
+"""
+
+from __future__ import print_function
+import argparse
+import errno
+import fcntl
+import os
+import stat
+import sys
+
+
+SPARSE_FILES_DIR = '/var/lib/metalk8s/storage/sparse'
+
+# https://github.com/torvalds/linux/blob/master/Documentation/admin-guide/devices.txt#L191
+LOOP_MAJOR = 7
+
+# https://github.com/torvalds/linux/blob/master/include/uapi/linux/loop.h#L110
+LOOP_CLR_FD = 0x4C01
```

Comment:
Since those constants are always used by a single function, maybe it's better to declare them near their usage.

Reply:
Good point, I just store them at the top of my modules, out of habit :)

Reply:
Hah, bike-shedding: I like them on top, actually.

Reply:
Ah sorry, missed your comment @NicolasT so already did the change. I think both sides have good arguments, but the stakes are too low for us to have this debate. So I won't change it anymore :stuck_out_tongue_winking_eye: